webui/react/src/components/DeterminedAuth.tsx

```diff
@@ -16,6 +17,9 @@ interface FromValues {
   username?: string;
 }
 
+const storage = new Storage({ basePath: '/DeterminedAuth', store: window.localStorage });
+const LAST_USERNAME = 'lastUsername'; // key for storing last det username
```

Comment:
suggestion: naming the constant to `STORAGE_KEY_USERNAME` and dropping the comment